internal/config/app.go

```diff
@@ -59,6 +59,12 @@ type RoutingVroomConfig struct {
 	Host string `mapstructure:"host"`
 }
 
+type MapConfig struct {
+	Center         []float64 `mapstructure:"center"`
+	BoundSouthWest []float64 `mapstructure:"bound_south_west"`
+	BoundNorthEast []float64 `mapstructure:"bound_north_east"`
```

Comment:
Same here, I would prefer an simple array
```suggestion
	BoundBox []float64 `mapstructure:"bbox"`
```